lib/services/http-service.js

```diff
@@ -142,6 +144,17 @@ function httpServiceFactory(
             });
         }
 
+        //Swagger support
+        var swaggerConfig = {
+            appRoot: path.resolve(__dirname, '../../'),
+            configDir: path.resolve(__dirname, '../../swagger_config'),
+            swagger: path.resolve(__dirname, '../../static/monorail-2.0.yaml'),
```

Comment:
@benbp - Thanks! I made the changes...
@WangWinson Thanks for the feedback. Since we have additional work to do on the swagger integration, we'll consider this for future PRs.